### ISSUE TITLE
Clean labels of training set

### ISSUE THREAD

**Issue by cissoidx** [google-research/reassessed-imagenet#3]
Hi, thanks for sharing. In the paper, there is 10 fold method to remove training images. Is there any plan to release the `Clean` labels of the training set?

**Comment by lucasb-eyer** [comment]
Hi and sorry for the late answer. If you look at the bottom of the README, they are actually linked there! The section is called [List of ImageNet training files](https://github.com/google-research/reassessed-imagenet#list-of-imagenet-training-files)

**Comment by lucasb-eyer** [comment]
(leaving the issue open for visibility)
